packages/in_app_purchase/in_app_purchase_android/android/src/main/java/io/flutter/plugins/inapppurchase/Translator.java

```diff
@@ -240,6 +241,16 @@ static HashMap<String, Object> fromBillingConfig(
     return info;
   }
 
+  /**
+   * Converter from {@link BillingResult} and {@link AlternativeBillingOnlyReportingDetails} to map.
+   */
+  static HashMap<String, Object> fromAlternativeBillingOnlyReportingDetails(
+      BillingResult result, AlternativeBillingOnlyReportingDetails details) {
+    HashMap<String, Object> info = fromBillingResult(result);
+    info.put("externalTransactionToken", details.getExternalTransactionToken());
```

Comment:
This just has to match the field name on the Dart side due to JSON generation, right? (I.e., it's expected that I don't see a matching string constant in Dart?)

Reply:
Yes that is my understanding and what happened with "countryCode" in my last pr that was tested end to end.